### ISSUE TITLE
WOW!

### ISSUE THREAD

**Issue by paulmsmith** [basaltinc/bedrock#147]
Hello,
I just wanted to be one of the first to say this looks so good! Majorly impressed. I wondered if you would like a team to kick the tyres of this? I'm assuming it's still in heavy development?

My initial impressions are this feels like a mixture of Fractal and Storybook with other lovely features added in, which is by way of saying, this is totally awesome and keep up the good work!

I was tempted to tweet this but having been working on things like this in the past, and not quite ready for a lot of noise (well intentioned as it may be), I wonder if you'd rather I hold off from saying how excited I am about this?

**Comment by paulmsmith** [comment]
Sure thing. I might tease it on Twitter.. "Watch for an exciting announcement from a design systems team at Clarity". Give my best to Jina (she's ace!).

I stumbled on this because I was looking for a good Node Twig implementation and remember talk on the Pattern Lab repo from @EvanLovely. I also worked in Government on a PEGA solution and remembered @sghoweri was working on Bolt.

All the best for Clarity, I think you're about to get a lot of positive attention. 👍
